ffi/src/lib.rs

```diff
@@ -862,7 +860,7 @@ pub extern "C" fn fwd_start_metrics_with_exporter(metrics_port: u16) -> Value {
 /// A `Value` containing {0, "error message"} if unable to get the latest metrics.
 #[unsafe(no_mangle)]
 pub extern "C" fn fwd_gather() -> Value {
-    Value::from(metrics_setup::gather_metrics().as_bytes())
+    metrics_setup::gather_metrics().map_or_else(Into::into, |s| Value::from(s.as_bytes()))
```

Comment:
This is a little awkward, but it seems that this was the pattern previously. Would have preferred adding `use metrics_setup::{gather_metrics, gather_metrics_with_exporter}` earlier in the file, and then:
```suggestion
    gather_metrics().map_or_else(Into::into, |s| Value::from(s.as_bytes()))
```

Reply:
You could also do `|s| s.as_bytes().into()`, right?

Reply:
Great point, modified to use `into()` instead of `from()`: 8c2c33a